impeller/fixtures/BUILD.gn

```diff
@@ -78,6 +78,7 @@ test_fixtures("file_fixtures") {
     "table_mountain_py.png",
     "table_mountain_pz.png",
     "test_texture.frag",
+    "flutter_logo_baked.png",
```

Comment:
nit: alphabetize.

Reply:
The formatter apparently only sorts DEPS and sources since it can't tell if the items in the custom template are order sensitive. [There is a NOSORT directive to disable sorting](https://gn.googlesource.com/gn/+/refs/heads/main/src/gn/command_format.cc#399). Maybe we should patch GN to add a SORT directive to the formatter.